rlbot_gui/story/story_challenge_setup.py

```diff
@@ -119,6 +120,23 @@ def rlbot_to_player_config(player: dict, team: Team):
     return player_config
 
 
+def script_to_script_config(script: dict):
```

Comment:
We don't seem to use any other properties in the dict so can we just directly accept the path instead?